### ISSUE TITLE
No Temp on Raspberry Pi 3 (Jessie) with Python 3.6 sideloaded

### ISSUE THREAD

**Issue by tlefeuvre** [aristocratos/bpytop#208]
Hi,
First, thanks for this wonderfull tool ;-)
Working great on my brand new raspberry 4.
Still I have a problem where the CPU temp is not showing on my old raspberry pi 3 (Jessie) with python 3.6 sideloaded.

Reproducing the problem, taken from another bug report regarding the same temp problem, I ran:
```
python3.6 -c "import subprocess; print(round(float(subprocess.check_output(['vcgencmd', 'measure_temp'], text=True).strip()[5:-2])))"
```
and get

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/local/lib/python3.6/subprocess.py", line 403, in run
    with Popen(*popenargs, **kwargs) as process:
TypeError: __init__() got an unexpected keyword argument 'text'
```

removing text=True and the script returns the temperature as expected :

```
python3.6 -c "import subprocess; print(round(float(subprocess.check_output(['vcgencmd', 'measure_temp']).strip()[5:-2])))"
54
```

Thanks !

Thomas

**Comment by aristocratos** [comment]
@tlefeuvre 
Hi, and thank you!

Seems like the `text` parameter for `subprocess` wasn't added until python 3.7, should work with: `python3.6 -c "import subprocess; print(round(float(subprocess.check_output(['vcgencmd', 'measure_temp'], universal_newlines=True).strip()[5:-2])))"`, just to make sure we don't get binary data returned.

And we can hope `universal_newlines` doesn't get removed in a future python version :P

Will fix in next update.

**Comment by aristocratos** [comment]
@tlefeuvre 
Fixed in v1.0.51

**Comment by tlefeuvre** [comment]
> @tlefeuvre 
> Fixed in v1.0.51

Testing this asap. Thanks a lot !
